superset/views/csv_import.py

```diff
@@ -337,9 +355,9 @@ def _get_existing_database(self, database_id: int, schema=None) -> models.Databa
         schema -- the schema to be used
 
         Raises:
-            ValueError: 1. If the schema is not allowed for csv upload
-                        2. If the database ID is not valid
-                        3. If there was a problem getting the schema
+            SchemaException: 1. If the schema is not allowed for csv upload
+                             2. If the database ID is not valid
```

Comment:
Wouldn't that be a problem with the database ID and not the schema? in that case the Exception name is misleading in my opinion